client/state/countries/reducer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { SMS, DOMAIN, PAYMENT } from './constants';
+import {
+	COUNTRIES_RECEIVE,
+	COUNTRIES_REQUEST,
+	COUNTRIES_REQUEST_FAILURE,
+	COUNTRIES_REQUEST_SUCCESS
+} from 'state/action-types';
+import { countriesSchema } from './schema';
+import { createReducer } from 'state/utils';
+
+export const items = createReducer( { [ SMS ]: [], [ DOMAIN ]: [], [ PAYMENT ]: [] }, {
+	[ COUNTRIES_RECEIVE ]: ( state, action ) => ( { ...state, [ action.listType ]: action.countries } )
+}, countriesSchema );
+
+export const isFetching = createReducer( { [ SMS ]: false, [ DOMAIN ]: false, [ PAYMENT ]: false }, {
```

Comment:
Maybe
```
combineReducers( {
//...
    isFetchingSMS,
    isFetchingDomain,
    isFetchingPayment
```

Alternatively you can create subfolders with the standard items/isFetching reducers:
```text
countries/
├── SMS/
│   ├── reducer.js
├──Domain/
│   ├── reducer.js
└── Payment/
    └── reducer.js
```